testsuite/gnat2goto/tests/arrays_write_element/test.out

```diff
@@ -1,4 +1 @@
-[precondition_instance.1] memcpy src/dst overlap: SUCCESS
-[precondition_instance.2] memcpy source region readable: SUCCESS
-[precondition_instance.3] memcpy destination region writeable: SUCCESS
 VERIFICATION SUCCESSFUL
```

Comment:
Why have these expected assertion successes been removed? What does the arrays_write_element test actually assert, and why are the assertion successes/etc not expected in the output? Or is this whole test now superseeded by other tests?

Reply:
@chrisr-diffblue Assertions were removed because the goto function no longer contains the memcpy call, so no checks are generated for it. The test does not assert anything, just tries to write into an array using an index (CBMC never gets in symex). And yes: it is contained in other tests.